sdks/java/io/parquet/src/main/java/org/apache/beam/sdk/io/parquet/ParquetIO.java

```diff
@@ -135,6 +135,16 @@
  * ...
  * }</pre>
  *
+ * <p>Reading with projection can be enabled with the projection schema as following. The
+ * projection_schema contains only the column that we would like to read and encoder_schema contains
+ * all field but with the unwanted columns changed to nullable.
+ *
```

Comment:
It would be also great if we could mention what is the expected improvement by projecting columns such as better memory usage or faster reading time.

Reply:
Added

Reply:
The description is still a bit vague. Can you please elaborate more on how much improvement users could expect from the column projection? For example, if the transform with the column projection only reads a half of the columns, does it also use only half of the memory and finish two times faster than the transform without the column projection? (If not, why is that?)

Reply:
The improvement is not as significant, since the processing time saved is only the time to read the unwanted columns, the reader will still go over the entire data set since data for each column in a row is stored interleaved. So the test showed that there is only a slight reduction in the reading time. For the memory, it is hard for me to track with the Dataflow server.

Reply:
Please also adding 
```
Note that the improvement is not as significant though, since the processing
time saved is only the time to read the unwanted columns, the reader will still
go over the entire data set since data for each column in a row is stored interleaved.
```
to the comments. It will help users a lot to understand how the column projection works.

Reply:
Added